context/app/static/js/shared-styles/sections/SectionFooter/SectionFooter.stories.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import SectionFooter from './SectionFooter';
+
+export default {
+  title: 'Sections/SectionFooter',
+  component: SectionFooter,
+};
+
+export const Basic = {
+  args: {
+    items: [
+      { key: '1', component: <>Item 1</> },
+      { key: 2, component: <>Item 2</> },
```

Comment:
Not a big deal, but may be a good idea to have consistency between the `key` prop types

Reply:
Thanks!